engine/standalone/engine/src/main/scala/pl/touk/nussknacker/engine/standalone/FutureBaseStandaloneScenarioEngine.scala

```diff
@@ -0,0 +1,29 @@
+package pl.touk.nussknacker.engine.standalone
+
+import pl.touk.nussknacker.engine.Interpreter.FutureShape
+import pl.touk.nussknacker.engine.baseengine.TestRunner
+import pl.touk.nussknacker.engine.baseengine.TestRunner.EffectUnwrapper
+import pl.touk.nussknacker.engine.baseengine.capabilities.FixedCapabilityTransformer
+
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+object FutureBaseStandaloneScenarioEngine {
+
+  type InterpreterType = StandaloneScenarioEngine.StandaloneScenarioInterpreter[Future]
+  private val scenarioTimeout = 10 seconds
+
+  implicit val cap: FixedCapabilityTransformer[Future] = new FixedCapabilityTransformer[Future]()
+
+  implicit val unwrapper: EffectUnwrapper[Future] = new EffectUnwrapper[Future] {
```

Comment:
Is it "final" solution that this uwrapper is created statically? What about timeout configuration? If not, maybe some TODO?

Reply:
Added TODO, though this timeout was fixed also in the previous solution. It's for test only